src/tests/trip/view-comments.test.js

```diff
@@ -0,0 +1,61 @@
+import chai from 'chai';
```

Comment:
Put this in a comment folder instead since the comment feature has a controller.